### DISCUSSION TITLE
Vitest picks the wrong index from prisma package.json

### DISCUSSION THREAD

**Discussion by garth** [vitest-dev/vitest#2088] [Q&A] [answered]
Prisma generates a package.json like this:

```json
{
  "name": ".prisma/client",
  "main": "index.js",
  "types": "index.d.ts",
  "browser": "index-browser.js"
}
```

Vitest is configured for node testing:

```ts
/// <reference types="vitest" />
import { defineConfig } from 'vite'

export default defineConfig({
  test: {
    environment: 'node',
  },
  esbuild: {
    platform: 'node',
  },
})
```

When running unit tests vitest picks the `browser` entry point from the generated prisma package.json.

Is this behaviour by design, I would have expected it to pick the index pointed to by `main` when the node environment is configured?

**Comment by sheremet-va** [comment, marked as answer]
This is Vite bug, it will be fixed in the next Vite and Vitest version

**Reply by PiotrJander** [reply]
Is there an issue to track this?
